test/powershell/Modules/Microsoft.PowerShell.Utility/Pester.Commands.Cmdlets.Json.Tests.ps1

```diff
@@ -1458,9 +1475,9 @@ Describe "Json Bug fixes"  -Tags "Feature" {
     It "ConvertFrom-Json deserializes an array of PSObjects (in multiple lines) as a single string." {
 
         # Create an array of PSCustomObjects, and serialize it
-        $array = [pscustomobject]@{ objectName = "object1Name"; objectValue = "object1Value" },
-                 [pscustomobject]@{ objectName = "object2Name"; objectValue = "object2Value" }
-
+        $array = [pscustomobject]@{ objectName = "object1Name"; objectValue = "object1Value" },
```

Comment:
Yes. But I believe this is because the source newlines were inconsistent. I didn't apply any changes here nor in the next one. IMHO better for newlines to be consistent than to have to manually reset them when tooling does the right thing.